src/features/pastelID/pastelIDSlice.ts

```diff
@@ -97,20 +102,32 @@ export function fetchPastelIDs(config: TRPCConfig): AppThunk {
 
 export function createPastelID(
   passphrase: string,
+  address: string,
   config: TRPCConfig,
 ): AppThunk {
   return async dispatch => {
     try {
       dispatch(createPastelIDStart())
-      const pastelid = await createNewPastelID(passphrase, config)
-      dispatch(createPastelIDSuccess(pastelid))
+      const res = await createNewPastelID(passphrase, address, config)
+      dispatch(
+        createPastelIDSuccess({
+          pastelid: res.pastelid,
+          txid: res.txid,
+        }),
+      )
+      dispatch(
+        openErrorModal({
+          title: 'PastelID has been created!',
+          body: `<div>PastelID: ${res.pastelid}<br />TXID: ${res.txid}</div>`,
```

Comment:
This should be just a string or a component, not a plain html

Reply:
I have updated for this.